Russian Culture

palfourti opened the pull request.

Viridianus reviewed (changes requested):
Change "Tsarevich" to "Tsarevna" for female while you're at it.

palfourti commented:
> 
> 
> Change "Tsarevich" to "Tsarevna" for female while you're at it.

Done, I guess you weren't the only one to miss that

Gr770 commented:
I recommend getting this approved by Viridianus before the merge. Something feels off, I don't think we should use "prince" for independents for Russian.

palfourti commented:
> 
> 
> I recommend getting this approved by Viridianus before the merge. Something feels off, I don't think we should use "prince" for independents for Russian.

If prince is being used for independents that's a mistake on my part; it should only be used for the children of rulers.

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
+	tier = county
+	cultures = { russian  }
+	governments = { tribal_government }
+	only_independent = yes
```
Is there any particular reason why tribals are given the independent tag?

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
+	tier = duchy
+	cultures = { russian  }
+	governments = { tribal_government }
+	only_independent = yes
```
Similar question to above

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { russian ilmenian severian volhynian }
```
I think using the whole cultural group, instead of listing all Induvidual cultures should be used for readability and debugging puporses

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+baron_female_russian = {
```
Changed the name from russian to eastern_slavic_group if you are going apply it to the whole culture. This is misleading

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+queen_mother_russian = {
```
Just a curious question, does queen_mother not apply to the empire tier?
 Also shouldn't it be ```top_liege = yes``` for this?

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
+	priority = 40
+	cultures = { russian }
+	governments = { feudal_government tribal_government }
+	top_liege = no
```
I think ```top_liege = yes``` (or just don't use it, its the default value) should be use for prince and princess.

Charanraja123 reviewed (commented):
There are some changes that need to be addressed. I've commented them out

Charanraja123 commented:
You haven't used the clan government anywhere. I personally would suggest adding clan to everywhere feudal is applied due to their similarity 
This is totally up for discussion. And i think its better if we set a standard about this

palfourti commented:
The first two comments are indeed oversights on my part and should be fixed by me. As for the rest, I'm following what I believed to be the precedent set by other titles that had been added before I started contributing (for instance, "king...norman" also includes a whole host of other cultures), though I will admit those titles didn't share the same culture group unlike these titles so perhaps a little independent thought was in order. 

Clan government is definitely something that should be discussed because I don't believe I added it to any of the cultures I've added and cultures other people added are missing it as well.

Charanraja123 commented:
> 
> 
> The first two comments are indeed oversights on my part and should be fixed by me. As for the rest, I'm following what I believed to be the precedent set by other titles that had been added before I started contributing (for instance, "king...norman" also includes a whole host of other cultures), though I will admit those titles didn't share the same culture group unlike these titles so perhaps a little independent thought was in order.
> 
> Clan government is definitely something that should be discussed because I don't believe I added it to any of the cultures I've added and cultures other people added are missing it as well.

Sorry if my comments felt targeted at you. I did not intend to blame you in particular, It was more of a general comment on our practices (myself included). I think its only at this point that all of us have a good enough understanding on the project to correct each other mistakes. I was just trying to question everything, and make sure we set the right precedent for future edits. 

I'll open an issue tab regarding how to handle Clan governments and others we might have left out

palfourti commented:
Oh no, I don't feel targeted, you're fine. Obviously those are things we should get settled so they don't have to be go back and changed later if we develop a more rational system for adding titles.

palfourti commented:
Suggested changes have been made. If everything looks right with them I think it's ready.

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,399 @@ duke_tribal_female_italian = {
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { east_slavic_group }
```
I do not know if this works. 
In the vanilla, culture groups are defined using the condition
`culture_groups = { foo bar }`

Viridianus replied:
If this works, it's ready.

palfourti replied:
> 
> 
> I do not know if this works.
> In the vanilla, culture groups are defined using the condition
> `culture_groups = { foo bar }`

Oh, I didn't know that. Fixed.

Charanraja123 commented on common/flavorization/00_CTP.txt:
```diff
@@ -6526,6 +6526,399 @@ duke_tribal_female_italian = {
+	gender = male
+	special = ruler_child
+	tier = empire
+	priority = 40
```
The title_prince__empire has a priority lesser than king_male_russian. This mean that the king title is usually preferred over the title prince__empire. If this is intended, then its okay

Viridianus replied:
This should be intended; it would be weird as hell to see "tsarevich" instead of the king title.

Gr770 commented:
Conflicts resolves, small tier fix. MERGING
